addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -286,15 +287,16 @@ public void packetReceived(byte[] packet) {
                     logger.debug("Transmitter response received: {}", resp);
 
                     transmitQueue.sendNext();
-                } else {
-
+                } else if (message instanceof RFXComDeviceMessage) {
                     for (DeviceMessageListener deviceStatusListener : deviceStatusListeners) {
                         try {
-                            deviceStatusListener.onDeviceMessageReceived(getThing().getUID(), message);
+                            deviceStatusListener.onDeviceMessageReceived(getThing().getUID(), (RFXComDeviceMessage)message);
                         } catch (Exception e) {
                             logger.error("An exception occurred while calling the DeviceStatusListener", e);
                         }
                     }
+                } else {
+                    // TODO handle this case
```

Comment:
There is a TODO, what should be done and can it be completed before merging the PR?